content/news/XX-XX-XXXX-mixxx_2_4_features.md

```diff
@@ -0,0 +1,83 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+
+## Mixxx 2.4
+
+Mixxx 2.4 is our next major release shipping the work of 2 years continuous development on features, code refacturing and bug fixing.
+
+### EngineOS export
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine OS devices ( Denon and Numark standalone controllers).
+This allows users to prepare their tracks on a laptop using Mixxx, export them on an USB stick and plug it to the standalone controller at your gig.
+
+### Saved Loops
+Mixxx 2.4 adds support for Saved loops
+If you have already tracks with Saved Loops defined by Serato DJ, Mixxx 2.4 will automatically read them from the track metadata.
+
+
+### Effects
+Mixxx 2.4 introduces Effect Chains, a set of effects with parameters which can be saved and loaded with all it's parameters. Mixxx 2.4 visualizes these effect parameter by value, unit and knob position.
+
+### Sync
+Mixxx 2.4 adds support for setting an explicit leader for sync. Only one deck can be the sync leader, this is set and indicated by the new crown symbol button.
+This mode is useful if you work with tracks, with variable BPM.
```

Comment:
maybe at another sentence that summarizes the behavior. like "when multiple tracks have variable BPM and sync enabled, all the decks will follow the changing bpm of the sync leader" (obviously we'd need a little more work on the wording here).

Reply:
I think this mode becomes already relevant with one deck in variable BPM beatgrid.

Reply:
not sure, haven't looked at the details too closely yet since I have all tracks with const beatgrid anyways.

Reply:
I wrote this feature so I tweaked the text